bokun.yaml

```diff
@@ -5987,51 +6050,101 @@ definitions:
+        items:
+          type: string
+      passValidForDays:
+        type: integer
+        description: 'Specifies after how many days from the purchase the pass expires.Only used if Activity.bookingType == PASS and Activity.passExpiryType == RELATIVE_DATE.'
       pickupSelectionType:
+        description: 'Specifies how to handle pick up for this rate. NB: if Activity.meetingType == MEET_ON_LOCATION, then this will always be UNAVAILABLE.'
         $ref: '#/definitions/AddonSelectionType'
       pickupPricingType:
+        description: 'Specifies whether pickup is included in the price or whether it is priced separately.'
         $ref: '#/definitions/AddonPricingType'
       pickupPricedPerPerson:
         type: boolean
+        description: 'Specifies whether pick up is priced per person (per pricing category). Only checked if Activity.meetingType != MEET_ON_LOCATION and pickupPricingType == PRICED_SEPARATELY.'
       dropoffSelectionType:
+        description: 'Specifies how to handle dropoff for this rate. Only used if Activity.offersDropoffService == true.'
         $ref: '#/definitions/AddonSelectionType'
       dropoffPricingType:
+        description: 'Specifies whether dropoff is included in the price or whether it is priced separately.'
         $ref: '#/definitions/AddonPricingType'
       dropoffPricedPerPerson:
         type: boolean
+        description: 'Specifies whether drop off is priced per person (per pricing category). Only checked if Activity.offersDropoffService == true and dropoffPricingType == PRICED_SEPARATELY.'
+      extraConfigs:
+        type: array
+        description: 'The list of activity extras that are used by this rate.'
+        items:
+          $ref: '#/definitions/ActivityRateExtraConfig'
       startTimeIds:
         type: array
+        description: 'Only used if bookingType == DATE_AND_TIME. The list of IDs of activity start times that this rate is applicable for. If empty, then the rate will be available for all start times.'
         items:
           type: integer
           format: int64
+      tieredPricingEnabled:
+        type: boolean
+        description: 'Specifies if there is tiered pricing set for this rate.'
+      tiers:
+        type: array
+        description: 'The list of pricing tiers which are set for this rate.'
+        items:
+          type: object
+          properties:
+            minPassengersRequired:
+              type: integer
+            maxPassengersRequired:
+              type: integer
+            pricingCategoryId:
+              type: integer
+              format: int64
+            activityRateId:
+              type: integer
+              format: int64
       pricingCategoryIds:
         type: array
+        description: 'The list of the IDs of pricing categories that are mapped to this rate.'
         items:
           type: integer
           format: int64
-      extraConfigs:
+      details:
```

Comment:
Has `extraConfigs` disappeared or was it moved to another line?

Reply:
moved